tcms/locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -49,6 +49,21 @@ msgstr "URL"
 msgid "Comment"
 msgstr "Bemerkung"
 
+#: tcms/core/contrib/linkreference/tests.py:84
+msgid "Enter a valid URL."
+msgstr ""
```

Comment:
Use `make messages` to regenerate .po files. They should exclude translations in tests.